roles/pulp_common/tasks/install_pip.yml

```diff
@@ -250,6 +242,13 @@
         clients: "{{ pulp_install_dir }}/bin/pip"
       register: pip_pkgs
 
+    - name: Install patched versions of dependencies (FIPS only)
+      shell: |
+        django_location=$({{ pulp_install_dir }}/bin/pip3 show Django | grep Location | cut -d' ' -f2)
+        sed -i 's/hashlib.md5()/hashlib.md5(usedforsecurity=False)/' "$django_location/django/db/backends/utils.py"
+      when: ansible_fips
+      failed_when: "'pulp_devel' not in role_names"
```

Comment:
This seems a bit problematic to me. But i see, that it must probably live in the middle of the pulp_common role.
Can we add an additional assertion that ansible_fips cannot be used without pulp_devel before any reasonable task is started?
Somewhere around here https://github.com/pulp/pulp_installer/blob/master/roles/pulp_common/tasks/main.yml#L13 ?

Reply:
Also +1 to implementing this failing early suggestion.